src/TikaOnDotnet.TextExtractor/TextExtractorOCR.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using java.io;
+using org.apache.tika;
+using org.apache.tika.config;
+using org.apache.tika.detect;
+using org.apache.tika.metadata;
+using org.apache.tika.mime;
+using org.apache.tika.parser;
+using org.apache.tika.parser.ocr;
+using org.apache.tika.parser.pdf;
+using Exception = System.Exception;
+using Object = java.lang.Object;
+using Parser = org.apache.tika.parser.Parser;
+using StringBuilder = System.Text.StringBuilder;
+
+namespace TikaOnDotNet.TextExtraction
+{
+    /// <summary>
+    ///     With this class text can be extracted from all different kind of documents with
+    ///     the use of Tika and Tesseract
+    /// </summary>
+    public class TextExtractorOCR : TextExtractor
+    {
+        #region Fields
+        private TesseractOCRConfig _tesseractOcrConfig;
+        #endregion
+
+        #region Properties
+        /// <summary>
+        /// Set the <see cref="TesseractOCRConfig"/> and enables extraction of text from
+        /// images with the use of Tesseract
+        /// </summary>
+        /// <remarks>
+        /// See https://tika.apache.org/1.14/api/org/apache/tika/parser/ocr/TesseractOCRParser.html
+        /// </remarks>
+        public TesseractOCRConfig TesseractOCRConfig
```

Comment:
This config, since it is required by this type, should be an argument to the constrictor.